src/segment_language.go

```diff
@@ -43,6 +53,24 @@ func (l *language) string() string {
 		return l.props.getString(MissingCommandTextProperty, MissingCommandText)
 	}
 	l.setVersion()
+
+	// if regex found no version
+	if l.version == "" {
+		return ""
+	}
+
+	// build release notes hyperlink
+	if l.props.getBool(EnableHyperlink, false) {
+		// if url template is blank in the confi file
+		if l.urlTemplate == "" {
```

Comment:
this check can be in the initial if statement as the return below does the same.